ui/src/routes/guide.svelte

```diff
@@ -0,0 +1,141 @@
+		<p>In this page, the timeline is not interactive.<p>
+
+		<h3>Sidebar</h3>
+		<p>Here you find a list of indicators, grouped in various thematic sections.<p>
+		<p>Clicking on a title will navigate to the trend view for that indicator.<p>
+
+		<h2>Indicator trends</h2>
+
+		<h3>Timeline</h3>
+		<p>The Timeline shows one white dot for each available year. It is now interactive in that by clicking on a dot lets you navigate to the indicator yearly view for that year.<p>
+		<p>In this page, the timeline is not interactive.<p>
+
+		<h3>Sidebar</h3>
+		<p>The sidebar has the same function described above: you can use it to navigate to the trends page of the available indicators.<p></p>
+
+		<h3>Trends</h3>
+		<p>Here you find a set of trend lines, one for each selected NUTS2 regions.</p>
+		<p>Hovering trends, highlights the trend and the data point nearest to the pointer.</p>
+		<p>Please use the switch on the top-right corner to choose between showing trends with absolute or ranked values.</p>
+
+		<h3>Regional selection</h3>
+		<p>It is possible to select NUTS1 regions to inherently select its contained NUTS2 region. This is particularly useful because NUTS1 regions don't vary over the years whereas NUTS2 regions do.</p>
+		<p>By default all NUTS2 regions are selected. The "globe" button toggles the menu where you can select the NUTS1 regions.</p>
+		<p>The selected NUTS2 regions can be highlighted or filtered.</p>
+		<p>Highlighting is useful to focus the attention on the trends for the desired regions while still being able to see them all.</p>
+		<p>Filtering is useful to isolate trends for the desired regions: when filtering, the color scale on the left gets updated only considering the values of the active trends.</p>
+
+		<h3>indicator info and download</h3>
+		<p>Clicking in the "i" icon at the right of the page title shows an info panel reporting the indicator metadata (data provenance, date of processing, etc).</p>
+		<p>In that same panel there is a button to download the <a href={csvWikiURL}>CSV file</a> of the current indicator.</p>
+		<p>The panel cak be dismissed by clicking iot the page background.</p>
+
+		<h2>Indicator yearly data</h2>
+
+		<p>This page cak be reached either by clicking on a dot in the index page or by clicking on one of the dots of the timeline in the trend of a certain indicator.<p></p>
+
+		<h3>Sidebar</h3>
+		<p>The sidebar has the same function described above: you can use it to navigate indicators trend page.<p></p>
+
+		<h3>Map</h3>
+		<p>On the left there is a choropleth showing the indicator values on the map of UK NUTS2 regions.</p>
+		<p>Gray regions indicate that there is no data available for that region.</p>
+		<p>The map shows cities with the most research intensive universities (called "Trac peer group A"), as used previously on maps within BEIS because they are key to BEIS regional policies.</p>
```

Comment:
```suggestion
		<p>The map shows cities with the most research intensive universities (those in "TRAC peer group A"). This group has been chosen due to its key relevance for BEIS regional policies.</p>
```
BEIS might have a wording that they would prefer to use here.